test/sqllogictest/transform/relation_cse.slt

```diff
@@ -1444,3 +1444,94 @@ Used Indexes:
   - materialize.public.i1
 
 EOF
+
+
+# WITH MUTUALLY RECURSIVE support
```

Comment:
I tried that, but the syntax is very restricted (you cannot have `WITH` outside or inside the `body` of an outer `WITH` or `WITH MUTUALLY RECURSIVE`. I have a test in #18714, though (look for the `relation_cse.spec` and `anf.spec` files).

Reply:
> look for the relation_cse.spec and anf.spec files

I've now looked at those spec files, but somehow I can't find the nested WMR test. Could you please point me to the specific line?

> I tried that, but the syntax is very restricted (you cannot have WITH outside or inside the body of an outer WITH or WITH MUTUALLY RECURSIVE

You can definitely have a WMR inside the body of an outer WMR. This situation is referred to as "sequenced WITH MUTUALLY RECURSIVE" in `with_mutually_recursive.slt`. (Note that if a WMR appears anywhere inside the body of an outer WMR (not necessarily at the root), then it will be pulled to the root of the body by `NormalizeLets`.)

But I was thinking more of a test where a WMR appears in one of the `values`s of an outer WMR.